google/cloud/internal/async_polling_loop_test.cc

```diff
@@ -66,20 +66,18 @@ class MockPollingPolicy : public PollingPolicy {
 };
 
 AsyncPollLongRunningOperation MakePoll(std::shared_ptr<MockStub> const& mock) {
-  return
-      [mock](CompletionQueue& cq, std::unique_ptr<grpc::ClientContext> context,
-             google::longrunning::GetOperationRequest const& request) {
-        return mock->AsyncGetOperation(cq, std::move(context), request);
-      };
+  return [mock](CompletionQueue& cq, auto context,
```

Comment:
Do we need similar changes for cases like lines 51 and 56 in this file?

Reply:
Those come later.

In one big PR I will update the stub signatures in the generator / handwritten code, internal classes, and the signatures for these test mocks.